x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -147,6 +145,12 @@ export class EnterpriseSearchRequestHandler {
     };
   }
 
+  getBodyAsString(body: object | Buffer): string | undefined {
```

Comment:
Alternative to a comment block - we could document this in unit tests instead! Can we write some just for the `EnterpriseSearchRequestHandler.getBodyAsString` method?